src/deploy/function_events_invited.sql

```diff
@@ -4,32 +4,43 @@
 -- requires: schema_public
 -- requires: table_invitation
 -- requires: table_contact
+-- requires: table_account_block
 -- requires: role_account
 -- requires: role_anonymous
 
 BEGIN;
 
-CREATE FUNCTION maevsi_private.events_invited()
-RETURNS TABLE (event_id UUID) AS $$
+CREATE OR REPLACE FUNCTION maevsi_private.events_invited()
+  RETURNS TABLE(event_id uuid)
+AS $$
 DECLARE
   jwt_account_id UUID;
 BEGIN
   jwt_account_id := NULLIF(current_setting('jwt.claims.account_id', true), '')::UUID;
 
   RETURN QUERY
-  SELECT invitation.event_id FROM maevsi.invitation
+  SELECT event_id FROM maevsi.invitation
```

Comment:
For the future I recommend short aliases like here:
`SELECT i.event_id FROM maevsi.invitation i`

Reply:
All of maevsi's codebase tries to prevent using abbreviations if possible as each abbreviation adds to the knowledge required to build an understanding of the code. It's considered less work to read a word than to look for, remember and recall the meaning of an abbreviation, especially as this project should stay as welcoming as possible to new developers.

I see that this kind of abbreviation you suggest is often used. I just want to not have the future discussions with others about "if we use abbreviations here, why not in another place as well" that I'm sure I'll be having... So let's not use abbreviations if there is not a requirement for them. Do you see such a requirement?